lang/lua-struct/Makefile

```diff
@@ -0,0 +1,32 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lua-struct
+PKG_VERSION:=0.9.2
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/iryont/lua-struct.git
+PKG_SOURCE_VERSION:=a86046034c28ed1a85bd4f2a877b903f232ea3d6
+PKG_MIRROR_HASH:=9d0deb73b6a42b0b85a8c271fea75862c7961ee468fad76700cf09e941ed934f
```

Comment:
Why do we need to add package, which is archived and thus only read-only for about an year?

No.

Reply:
See other comment.